app/javascript/util/filterUtil.tsx

```diff
@@ -0,0 +1,10 @@
+export const stripMostTags = (input, allowed?: string) => {
```

Comment:
leave a note that this comes from the angular version, and a TODO to seek out a better method for dealing with raw html